components/email/test/editorial_signup.coffee

```diff
@@ -105,6 +128,7 @@ describe 'EditorialSignupView', ->
         ARTICLE: channel_id: '123'
         ARTSY_EDITORIAL_CHANNEL: '123'
         SUBSCRIBED_TO_EDITORIAL: false
+      @view.ctaBarView = {close: sinon.stub()}
```

Comment:
Need to reset `@ctaBarView` here because we're calling `onSubscribe` directly, instead of letting `initialize` fill the value of `@ctaBarView`